### ISSUE TITLE
Can't run

### ISSUE THREAD

**Issue by bonemouth** [d-evelopers/HellsAngel#2]
Hi, I'm new to running something through Node.js, so maybe I'm doing something wrong, but I followed your directions and get this series of errors when I try to start: https://pastebin.com/fXaHqf7K

**Comment by thingywhat** [comment]
Huh! No guarantees, but it kinda looks like you're running into [this](https://stackoverflow.com/questions/24721903/npm-cli-js-not-found-when-running-npm).

A lot of those posts seem to suggest setting environment variables, but since that's *kiiind of* annoying, I think the easiest way to fix this is to nuke node.js from "Add/Remove Programs" and then reinstall with the installer [here](https://nodejs.org/en/download/).

After that, in the folder `git clean -dfx`, and then `npm install` should give you all that you need.

**Comment by bonemouth** [comment]
That didn't seem to help. I also tried using "git clone https://github.com/d-evelopers/HellsAngel.git," which gives me:

'git' is not recognized as an internal or external command,
operable program or batch file.

**Comment by bonemouth** [comment]
I was navigating to the folder via a command prompt. I tried it this way and got the same error list, though. Do I need to download Git?

**Comment by thingywhat** [comment]
Nope! And actually I just noticed something in your pastebin:

```
c:\Program Files\nodejs>npm start
```

You don't wanna run that from `c:\Program Files\nodejs`, but instead from inside of HellsAngel's folder.

**Comment by bonemouth** [comment]
Right. I had originally put your game in there, but then I had the same thought. I tried it from the proper folder this time and got this error: https://pastebin.com/uni4PSkh.

**Comment by thingywhat** [comment]
Oh, this isn't a game. Right now it's actually just a VN engine with some tools to *make* a game that a few people were excited to build a while ago. (I guess I should update the readme!)

As for what you're running into, if this happens with a fresh install using the instructions above, that's actually *really*-weird.

Maybe `npm install -g npm` will make it play a little nicer? That should never be necessary, since you obviously already have npm installed, but that's what your errors are telling me. ¯\\_(ツ)_/¯
